src/Features/Core/Portable/DocumentationComments/IDocumentationCommentFormattingService.cs

```diff
@@ -11,5 +12,6 @@ internal interface IDocumentationCommentFormattingService : ILanguageService
     {
         string Format(string rawXmlText, Compilation compilation = null);
         IEnumerable<TaggedText> Format(string rawXmlText, SemanticModel semanticModel, int position, SymbolDisplayFormat format = null);
```

Comment:
who uses thsi form?  is it still needed?

Reply:
Shift-F12 doesn't return (like, takes ages) so i cannot tell you.
but i remember that i got lots of errors when i wanted to change the method declaration, initially. that's why i decided to introduce another overload.
...
i modified the signature to get errors: there are 38 places that want that method.

Reply:
there appear to be only one reference to this method.

Reply:
dunno why my last comment wasn't saved - i looked, and there were almost 40 places that referenced that overload.

Reply:
there are not.  i checked.  this is the only location that references this.  if you have other locations you think reference this, can you please link to them?  Thanks!

Reply:
yeah, after intellisense stopped burning cpu, the reference count was visible, too
i was mistaken. overload is gone.